pkg/doc/signature/jsonld/processor.go

```diff
@@ -27,42 +28,56 @@ var logger = log.New("aries-framework/json-ld-processor")
 // ErrInvalidRDFFound is returned when normalized view contains invalid RDF.
 var ErrInvalidRDFFound = errors.New("invalid JSON-LD context")
 
-// normalizeOpts holds options for canonicalization of JSON LD docs.
-type normalizeOpts struct {
-	removeInvalidRDF bool
-	validateRDF      bool
-	documentLoader   ld.DocumentLoader
-	externalContexts []string
+// processorOpts holds options for canonicalization of JSON LD docs.
+type processorOpts struct {
+	removeInvalidRDF    bool
+	validateRDF         bool
+	documentLoader      ld.DocumentLoader
+	externalContexts    []string
+	documentLoaderCache map[string]interface{}
 }
 
 // ProcessorOpts are the options for JSON LD operations on docs (like canonicalization or compacting).
-type ProcessorOpts func(opts *normalizeOpts)
+type ProcessorOpts func(opts *processorOpts)
 
 // WithRemoveAllInvalidRDF option for removing all invalid RDF dataset from normalize document.
 func WithRemoveAllInvalidRDF() ProcessorOpts {
-	return func(opts *normalizeOpts) {
+	return func(opts *processorOpts) {
 		opts.removeInvalidRDF = true
 	}
 }
 
 // WithDocumentLoader option is for passing custom JSON-LD document loader.
 func WithDocumentLoader(loader ld.DocumentLoader) ProcessorOpts {
-	return func(opts *normalizeOpts) {
+	return func(opts *processorOpts) {
 		opts.documentLoader = loader
 	}
 }
 
+// WithDocumentLoaderCache option is for passing cached contexts to be used by JSON-LD context document loader.
+func WithDocumentLoaderCache(cache map[string]interface{}) ProcessorOpts {
```

Comment:
I read it in the PR desc that string is supported as the value for cache map. Can we mention the supported types for the cache value in the godoc as well?